praw/objector.py

```diff
@@ -1,13 +1,19 @@
 """Provides the Objector class."""
+from .models.reddit.base import RedditBase
```

Comment:
I think this should go after the next line.

Reply:
Understood